cmake/Config.cmake

```diff
@@ -44,7 +44,20 @@ endif()
 # -DPython_ADDITIONAL_VERSIONS=Major.Minor
 # if your python version is not implicitly supported by cmake
 find_package (PythonInterp 2.7)
-find_package (PythonLibs 2.7)
```

Comment:
Based on that doc and the output of several different attempts, I don't think `find_package` works unless there is a `.cmake` available. The homebrew-provided `Python.framework` does not provide a `.cmake`.

How about running `find_package` and this code unconditionally and then setting `PYTHON_LIBRARY` if this code got a different answer?